.editorconfig

```diff
@@ -0,0 +1,21 @@
+# EditorConfig helps developers define and maintain consistent
```

Comment:
The [editorconfig standard](https://editorconfig.org/) helps maintain consistent formatting across editors and developers, in this case ensuring my editor didn't change your indentation from 4 spaces -> 2.